sass-lint.yml

```diff
@@ -0,0 +1,55 @@
+#########################
+## Starter config file
+#########################
+# Linter Options
```

Comment:
I just checked out the branch and ran the linter. Getting lots of warnings for things like `property-sort-order` and `class-name-format`. Not a big deal -- but maybe we can add a TODO to customize it a bit more to fit our style?

Is it doing what you need as far as calling out errors that will impact the build to React?